bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescription.java

```diff
@@ -113,7 +115,8 @@ public ConfigDescription(URI uri, List<ConfigDescriptionParameter> parameters,
      *
      * @return the URI of this description (not null)
      */
-    public URI getURI() {
+    @Override
+    public URI getUID() {
```

Comment:
@sjka I just found this one here, which leads to some broken bindings in OH2 repo (see [Travis failure here](https://travis-ci.org/openhab/openhab2-addons/builds/254991394?utm_source=github_status&utm_medium=notification)). Seems to be an APIBraking, isn't it?